frontend/public/components/app.jsx

```diff
@@ -145,7 +149,7 @@ class App extends React.PureComponent {
             <LazyRoute path="/cluster-health" exact loader={() => import('./cluster-health' /* webpackChunkName: "cluster-health" */).then(m => m.ClusterHealth)} />
             <LazyRoute path="/start-guide" exact loader={() => import('./start-guide' /* webpackChunkName: "start-guide" */).then(m => m.StartGuidePage)} />
 
-            <LazyRoute path="/marketplace" exact loader={() => import('./marketplace/kubernetes-marketplace' /* webpackChunkName: "marketplace" */).then(m => m.MarketplacePage)} />
+            <LazyOnceRoute path="/marketplace" exact componentName="MarketplacePage" loader={() => import('./marketplace/kubernetes-marketplace' /* webpackChunkName: "marketplace" */)} />
```

Comment:
Looks like the view of categories for ```/marketplace``` and ```/marketplace?categories=all``` is different. Maybe this should be ```/marketplace?categories=all``` ? Or it should reroute to it somehow? What do you think?

Reply:
There should be no difference. I don't see any. What are you seeing differently?

Reply:
Sounds good!